MAUI/PDF-Viewer/Magnification.md

```diff
@@ -47,3 +48,62 @@ PdfViewer.MaxZoomFactor = 2;
 N>If the `ZoomFactor` value is set beyond the [MinZoomFactor](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_MinZoomFactor) and [MaxZoomFactor](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_MaxZoomFactor) limits, then it will not be effective.
 
 The example project with the zoom or magnification functionalities can be downloaded [here](https://github.com/SyncfusionExamples/maui-pdf-viewer-examples).
+
+## Zoom modes 
+
+The .NET MAUI PDF Viewer supports the below zoom modes: 
+1. Fit to Page
+2. Fit to Width
+
+Fit-to-page ensures that users can see the entire page content to get a quick overview without having to scroll. Fit-to-width is helpful when reading documents with narrow columns, such as newspaper articles or other multicolumn layouts.
+You can change the zoom mode using the `SfPdfViewer.ZoomMode` property of the PDF Viewer, default value for `SfPdfViewer.ZoomMode` is `ZoomMode.Default`.
+### How to setting zoom mode from toolbar? 
```

Comment:
**How to setting** Need to check Grammar

Reply:
Feedback addressed